OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/createDomainAndStart.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+# # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please remove the comma after `2020` and the phrase "`All rights reserved.`" from all your copyright headers in all files.

Reply:
done